frontend/public/components/monitoring/alerting.tsx

```diff
@@ -585,180 +588,222 @@ const alertStateToProps = (state: RootState, { match }): AlertsDetailsPageProps
-                        >
-                          {_.get(rule, 'name')}
-                        </Link>
-                      </div>
-                    </dd>
-                  </dl>
-                </div>
-              </div>
-            </div>
+    const ele = (
+      <StatusBox data={alert} label={AlertResource.label} loaded={loaded} loadError={loadError}>
+        <h1 className="co-m-pane__heading">
+          <div data-test="resource-title" className="co-resource-item">
+            <MonitoringResourceIcon className="co-m-resource-icon--lg" resource={AlertResource} />
+            {labels?.alertname}
+            <SeverityBadge severity={labels?.severity} />
           </div>
-          {silencesLoaded && !_.isEmpty(alert?.silencedBy) && (
-            <div className="co-m-pane__body">
-              <div className="co-m-pane__body-group">
-                <SectionHeading text="Silenced By" />
-                <div className="row">
-                  <div className="col-xs-12">
-                    <Table
-                      aria-label="Silenced By"
-                      data={alert?.silencedBy}
-                      Header={silencesTableHeader}
-                      loaded={true}
-                      Row={SilenceTableRow}
-                    />
-                  </div>
-                </div>
-              </div>
+          {state !== AlertStates.Silenced && (
+            <div className="co-actions" data-test-id="details-actions">
+              <ActionButtons actionButtons={[silenceAlert(alert)]} />
             </div>
           )}
-        </StatusBox>
-      </>
+        </h1>
+        <HeaderAlertMessage alert={alert} rule={rule} />
+      </StatusBox>
+    );
+
+    const alertPodObj = {
+      namespace: alert?.labels.namespace,
+      kind: 'Pod',
+      kindObj: {
+        apiVersion: 'v1',
+        label: 'Pod',
+        labelKey: 'public~Pod',
+        plural: 'pods',
+        abbr: 'P',
+        namespaced: true,
+        kind: 'Pod',
+        id: 'pod',
+        labelPlural: 'Pods',
+        labelPluralKey: 'public~Pods',
+      },
+      name: alert?.labels.pod,
+      badge: null,
+    };
+
+    return (
+      <DetailsPage
+        match={match}
+        namespace={alertPodObj.namespace}
+        kind={alertPodObj.kind}
+        kindObj={alertPodObj.kindObj}
+        name={alertPodObj.name}
+        ele={ele}
+        breadcrumbsFor={() => [
+          {
+            name: t('public~Alerts'),
+            path: namespace ? `/dev-monitoring/ns/${namespace}/alerts` : '/monitoring/alerts',
+          },
+          { name: t('public~Alert details'), path: undefined },
+        ]}
+        pages={[
+          navFactory.details(() => (
+            <Details
+              alert={alert}
+              rule={rule}
+              namespace={alertPodObj.namespace}
+              silencesLoaded={silencesLoaded}
+            />
+          )),
+          navFactory.logs(AlertLogs),
+        ]}
+      />
```

Comment:
Also `queryParams` are required for fetching the alert and these were getting lost on clicking the tabs as horizontal-nav doesnt have support for adding `queryParams`.